hydrus/metadata/drone_doc_gen.py

```diff
@@ -14,15 +14,15 @@ def drone_doc(API, BASE_URL):
                        BASE_URL)
 
     # Status Class
-    status = HydraClass("Status", "Status", "Class for drone status objects")
+    status = HydraClass("http://hydrus.com/Status", "Status", "Class for drone status objects")
```

Comment:
The vocab has an `@id: <server_url>/api/vocab` we can use that to replace `vocab:`?